subprojects/docs/src/docs/userguide/software_model_concepts.adoc

```diff
@@ -28,3 +30,5 @@ The software model describes how a piece of software is built and how the compon
 * A _binary_ represents some output that is built for a component. A component may produce multiple different output binaries. For example, for a C++ library, both a shared library and a static library binary may be produced. Each binary is initially configured to be built from the component sources, but additional source sets can be added to specific binary variants.
 * A _variant_ represents some mutually exclusive binary of a component. A library, for example, might target Java 7 and Java 8, effectively producing two distinct binaries: a Java 7 Jar and a Java 8 Jar. These are different variants of the library.
 * The _API_ of a library represents the artifacts and dependencies that are required to compile against that library. The API typically consists of a binary together with a set of dependencies.
+
+The <<software_model_extend.adoc#software_model_extend, software model can be extended>> enabling deep modeling of specific domains via richly typed DSLs.
```

Comment:
Don't understand why this was added if the software model is being deprecated. Regardless, I think there should be a command between "extended" and "enabling".

Reply:
I couldn't find any link to extending the software model chapters. If we remove those chapter from the navigation bar, only users that know the direct link to the chapter will be able to browse it. As a previous software model user, the extending chapter was really useful.